aiproteomics/modelgen/save_model.py

```diff
@@ -0,0 +1,78 @@
+import os
+import tf2onnx
+
+def save_model(
```

Comment:
This is very nice now. The only thing I can think of is that we could maybe have type hints for the parameters (e.g. since we know that framework is a string etc)